src/main/java/kg/jarkyn/server/ReceivingSocket.java

```diff
@@ -0,0 +1,22 @@
+package kg.jarkyn.server;
+
+import kg.jarkyn.server.Incoming.Requester;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class ReceivingSocket {
+    private ServerSocket serverSocket;
+
+    public ReceivingSocket(ServerSocket serverSocket) {
```

Comment:
oh yeah, that is just a wrapper to make server testable without having to actually open sockets

this was the whole point of yesterday actually, took me ages to understand how to do this